### PR TITLE
added tab to unlock wallet by mnemonic

### LINKED ISSUE #507: [wallet] unlock by mnemonic phrase
in https://iotexscan.io/wallet, unlock with mnemonic phrase

### PR DESCRIPTION
**Pull request by gyanpgupta**
/kind backend-change
/kind ui-change

Fixes #507

### CODE REVIEW

File: src/shared/wallet/unlock-wallet.tsx [hunk 1]
```diff
@@ -70,6 +71,9 @@ class UnlockWallet extends React.Component<Props, State> {
           <Tabs.TabPane tab={t("unlock-wallet.by_private_key")} key="2">
             <UnlockByPrivateKey />
           </Tabs.TabPane>
+          <Tabs.TabPane tab={"By Mnemonic Phrase"} key="3">
```
Comment (puncsky) [inline src/shared/wallet/unlock-wallet.tsx]: translation

File: src/shared/wallet/unlock-by-mnemonic.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  State
+> {
+  public state: State = {
+    mnemonicPhrase: ""
+  };
+
+  public handleInputChange = (e: React.FormEvent) => {
+    const name: string = get(e, "target.name");
+    const value = get(e, "target.value");
+    // @ts-ignore
```
Comment (puncsky) [inline src/shared/wallet/unlock-by-mnemonic.tsx]: use correct type annotation instead

**Comment by gyanpgupta** [comment]
@puncsky , Updated, please have a look . thanks

File: translations/de.yaml [hunk 3]
```diff
@@ -413,7 +413,6 @@ account.actionHistory: Aktionsverlauf
 home.stats.tps: 'Aktuelle TPS / Peak'
 home.stats.candidates: IoTeX-Konsensdelegierte
 home.stats.productivity: 'Durchschnittliche Produktivität'
-home.stats.progressEpoch: Fortschritt der Epoche
```
Comment (nmhung1210) [inline translations/de.yaml]: Why remove this text? I was merged to the master before.

**Pull request merged**